ekan0ra/queue.py

```diff
@@ -27,13 +27,11 @@ def enqueue(self, nick):
 
     def dequeue(self, nick):
         """
-        Remove first item matching `nick` in queue.
+        Remove every occurrence of user from a queue.
         """
-        try:
-            self.remove(nick)
-            return True
-        except ValueError:
-            return False
+        result=self.count(nick)>0
```

Comment:
Please space your operators according to this [PEP guideline](http://pymbook.readthedocs.io/en/latest/pep8.html#other-recommendations).